ffi/diplomat/src/segmenter_line.rs

```diff
@@ -243,6 +246,10 @@ pub mod ffi {
         /// Finds the next breakpoint. Returns -1 if at the end of the string or if the index is
         /// out of range of a 32-bit signed integer.
         #[allow(clippy::should_implement_trait)]
+        #[diplomat::rust_link(icu::segmenter::LineBreakIterator, Struct)]
+        #[diplomat::rust_link(icu::segmenter::LineBreakIterator::into_iter, FnInStruct, compact)]
```

Comment:
nit: this is a default trait method i don't think we need to visibly link to it.